### PR TITLE
PM-1918 - Update Matterbridge bot base image

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

File: matterbridge-bot/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ helmfile status
 | fullnameOverride | string | `""` | The full release name override |
 | image.pullPolicy | string | `"IfNotPresent"` | The pullPolicy used when pulling the image |
 | image.repository | string | `"42wim/matterbridge"` | The repository of the image |
-| image.tag | string | `"latest"` | The tag of the image. Overrides the image tag whose default is the chart appVersion. |
+| image.tag | string | `"stable"` | The tag of the image. Overrides the image tag whose default is the chart appVersion. |
```
Comment (simisimis) [inline matterbridge-bot/README.md:48]: nit: If you're setting default to `stable` in values.yaml , then chart version will never default to chart appVersion

**Review by simisimis — APPROVED** [review]
👍

**Review by johnmarcou — APPROVED** [review]
LGTM

File: matterbridge-bot/README.md (hunk 1 above)
Comment (kaozenn) [reply matterbridge-bot/README.md:48]: Unfortunately, their repo only have a few dynamic tags

**Pull request merged**